<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>520</width><height>380</height></rect>
  </property>
  <property name="windowTitle">
   <string>View Balance</string>
  </property>

  <widget class="QWidget" name="centralwidget">



   <layout class="QVBoxLayout">

    <item><widget class="QLabel" name="labelTitle"><property name="text"><string>Select Account</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>

    <item><widget class="QComboBox" name="accountCombo"><property name="minimumHeight"><number>40</number></property></widget></item>

    <item><widget class="QPushButton" name="btnCheck"><property name="text"><string>Check Balance</string></property><property name="minimumHeight"><number>45</number></property></widget></item>

    <item><widget class="QLabel" name="labelBalance"><property name="text"><string></string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>

    <item><widget class="QPushButton" name="btnBack"><property name="text"><string>Back</string></property><property name="minimumHeight"><number>40</number></property></widget></item>

   </layout>

  </widget>
 </widget>

 <resources/>
 <connections/>
</ui>
